src/features/communication/communication.scss

```
@@ -0,0 +1,56 @@
+.communication {
+  &.content {
+    &.info-wrapper {
+      display: flex;
+      flex-direction: column;
+      gap: 26px;
+
+      & .column-2 {
```

```suggestion
      .column-2 {
```
Please, fix here and below